### PR TITLE
Image recognition module [SVHN]

### PR DESCRIPTION
**Pull request by martinoywa**
This PR contains code for the updated image recognition module.
Main changes are an `updated model`  which now can `recognize multiple numbers` from a single image. 
This is due to us only using one canvas to draw on in the write page.

Check tests and updated module files for more details.

**Note:** `To run tests`
Tests will not run in workflow because of how git-lfs handles large files. Technically the model doesn't exist to be used for inferencing. You'll need to run the tests locally.

- [x] Clone the repo
- [x] Checkout `CU-apyem5_Image-recognition-module_Martin-Oywa-New`
- [x] Download  the weights https://github.com/martinoywa/numbers-tutor/blob/CU-apyem5_Image-recognition-module_Martin-Oywa-New/app/ai/write/checkpoints/model-54000.pth
- [x] Store weights in `checkpoints/` folder.
- [x] Run pytest -v

### CODE REVIEW

**Comment by martinoywa** [comment]
Task linked: [CU-apyem5 Image recognition module](https://app.clickup.com/t/apyem5)

**Comment by martinoywa** [comment]
**Note:** Tests will not run in workflow because of how git-lfs handles large files. Technically the model doesn't exist to be used for inferencing. You'll need to run the tests locally.

File: app/ai/write/tests/test_write.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ def test_checkpoints_exists(self):
             exists.
             :return: True if exists else False
         """
-        checkpoint = Path('app/ai/write/checkpoints/checkpoint.pt')
+        checkpoint = Path('app/ai/write/checkpoints/model-54000.pth')
         self.assertTrue(True, os.path.isfile(checkpoint))
 
     def test_model_loading(self):
```
Comment (KelynPNjeri) [inline app/ai/write/tests/test_write.py:19]: Ensure this test passes.

**Review by KelynPNjeri — CHANGES_REQUESTED** [review]
Fix the one test, apart from that all is great.

**Review by KelynPNjeri — APPROVED** [review]
Everything seems OK. Merge.

**Comment by martinoywa** [comment]
> Everything seems OK. Merge.

Perfect!

**Pull request merged**